Gopkg.toml

```diff
@@ -42,7 +47,11 @@ required = [
   name = "github.com/spf13/cobra"
   version = "0.0.3"
 
-
 [[constraint]]
   branch = "master"
   name = "k8s.io/apimachinery"
```

Comment:
We should be using matching versions of code-generator and apimachinery... perhaps best to switch code-generator to master for now too, if we require master of apimachinery? /cc @BenTheElder